src/mainTemplate.json

```diff
@@ -489,51 +275,11 @@
         "lbBackEndPoolsAdded": {
             "backendPools": [
                 {
-                    "id": "[concat(resourceId('Microsoft.Network/loadBalancers','lb-influxdb'),'/backendAddressPools/LBBE')]"
+                    "id": "[resourceId('Microsoft.Network/loadBalancers/backendAddressPools', 'lb-influxdb', 'LBBE')]"
```

Comment:
I assume this is right, but I want to pick you brain about how this LB hookup works at some point because I don't grok it yet.

Reply:
Yeah its right -- i just took a line from another template that was connecting the API call which is not wrong -- but the best practice tooling flagged the use of **concat()** so i just refactored it